tox.ini

```diff
@@ -33,9 +33,11 @@ deps =
 commands =
    python -V
    coverage --version
-   {envbindir}/trial --version
+   python -m twisted.trial --version
    coverage erase
-   coverage run -p {envbindir}/trial {posargs:towncrier}
+   # coverage fails to run trial on Windows since it is trial.exe
+   # so just use --module.
+   coverage run -p --module twisted.trial {posargs:towncrier}
```

Comment:
May not make any difference but this could also be `python -m coverage ...`.  I expect the right command is on PATH thanks to tox ... but a few lines above it looks like you thought `python -m twisted.trial ...` was a good idea, so maybe it's a good idea for coverage too?

Reply:
The `-m twisted.trial` was for consistency with this line which switched as explained in the comment.  But sure, could do it for everything.  I'll look, or maybe another PR to switch everything if there are several unrelated places where `-m` would make sense.

Reply:
https://tox.readthedocs.io/en/latest/config.html#conf-allowlist_externals

Hmm, so tox only warns about outside access and doesn't, right there anyways, list an option to make that an error.  Anyways, there are several other places this would apply in general so I won't change them here.  I did update the comment to better represent coverage's perspective on the run subcommand.